### PR TITLE
feat: add validator blacklist

### PR DESCRIPTION
**Pull request by devwckd**
# Pull Request Checklist

Before submitting this PR, please make sure:

- [ ] You have run `cargo clippy` and addressed any warnings
- [ ] You have added appropriate tests (if applicable)
- [ ] You have updated the documentation (if applicable)
- [ ] You have reviewed your own code
- [ ] You have updated changelog (if applicable)

## Description

Please provide a brief description of the changes in this PR.

## Related Issues

Please link any related issues here

### CODE REVIEW

**Comment by devwckd** [comment]
@Supremesource or @saiintbrisson pls run benchmarks for me and fix the pallet calls if possible, I cant run them, thanks :D

File: pallets/subspace/src/subnet.rs [hunk 1]
```diff
@@ -563,4 +563,52 @@ impl<T: Config> Pallet<T> {
...
+        origin: T::RuntimeOrigin,
+        netuid: u16,
+        module: T::AccountId,
+    ) -> DispatchResult {
+        let key = ensure_signed(origin)?;
+
+        let params = Self::subnet_params(netuid);
+        ensure!(params.founder == key, Error::<T>::NotFounder);
+
+        ensure!(
```
Comment (Supremesource) [inline pallets/subspace/src/subnet.rs:577]: remove this check, you can blacklist it even when it's not registered

File: pallets/subspace/src/subnet.rs [hunk 2]
```diff
@@ -563,4 +563,52 @@ impl<T: Config> Pallet<T> {
...
+            Uids::<T>::get(netuid, &module).is_some(),
```
Comment (Supremesource) [inline pallets/subspace/src/subnet.rs:602]: remove it here as well

**Review by Supremesource — CHANGES_REQUESTED** [review]
Fix the ensures please. Also you need to check if the person calling both extrinsic is acutally a subnet owner of that netuid, you don't do this now

**Comment by devwckd** [comment]
> Fix the ensures please. Also you need to check if the person calling both extrinsic is acutally a subnet owner of that netuid, you don't do this now

@Supremesource  doesn't this work?

```
        let params = Self::subnet_params(netuid);
        ensure!(params.founder == key, Error::<T>::NotFounder);
```

**Comment by Supremesource** [comment]
> > Fix the ensures please. Also you need to check if the person calling both extrinsic is acutally a subnet owner of that netuid, you don't do this now
> 
> @Supremesource doesn't this work?
> 
> ```
>         let params = Self::subnet_params(netuid);
>         ensure!(params.founder == key, Error::<T>::NotFounder);
> ```

Oh, yeah, it does, I must have missed that

**Pull request merged**